ui/pages/send/send.container.js

```diff
@@ -77,27 +77,26 @@ function mapDispatchToProps(dispatch) {
     updateAndSetGasLimit: ({
       blockGasLimit,
       editingTransactionId,
-      gasLimit,
       gasPrice,
       selectedAddress,
       sendToken,
       to,
       value,
       data,
     }) => {
-      editingTransactionId
-        ? dispatch(setGasTotal(calcGasTotal(gasLimit, gasPrice)))
-        : dispatch(
-            updateGasData({
-              gasPrice,
-              selectedAddress,
-              sendToken,
-              blockGasLimit,
-              to,
-              value,
-              data,
-            }),
-          );
+      if (!editingTransactionId) {
```

Comment:
setGasTotal here is a useless method. getGasTotal just returns `gasLimit * gasPrice`